src/main/java/com/aws/greengrass/util/GreengrassServiceClientFactory.java

```diff
@@ -184,5 +204,6 @@ private void configureClient(DeviceConfiguration deviceConfiguration) {
             }
         }
         this.greengrassV2DataClient = clientBuilder.build();
+        this.greengrassV2DataClient.close();
```

Comment:
why are you building the client and instantly closing it?

Reply:
typo - forgot to remove this line when doing some other testing